bin/reportdaily.py

```diff
@@ -56,6 +64,142 @@ class MissingSubCommand(ValueError):
 log.addHandler(logging.NullHandler())
 
 
+def cmd_init(args):
+    """Creates an initial config file for a user config file"""
+
+    # create config by hand
+    check_if_config_exists(args)
+
+    log.debug("INIT selected %s", args)
+    print("Init selected", args)
+    return 0
+
+
+def create_config(args):
+    """This function will create a config file where the user data is stored"""
+    # time to ask the user for his data
+
+    # name
+    if args.name == None:
+        print("Please enter your full name --> example: 'Max Musterman'")
+        name = input("Ihr name ist ? ")
+    else:
+        name = args.name
+    # team
+    if args.team == None:
+        team = input("Please enter your team name: ")
+    else:
+        team = args.team
+    # year
+    if args.year == None:
+        year = int(input("What year of training are you in? "))
+    else:
+        year = args.year
+    # time
+    today_date = date.today()
+    print(
+        f"You are ready to go.  Entries for this {today_date} can  bee added now with reportdaily add")
+
+    # store configfiles in list
+    config_data = [name, team, today_date, year]
+
+    # create a config file
+    config = ConfigParser()
+    config["settings"] = {'name': name,
+                          'team': team, 'current_day': today_date, 'start_year': year}
+
+    # create a config file
+    os.makedirs(os.path.dirname(CONIFGPATH), exist_ok=True)
+    with open(CONIFGPATH, 'w') as user_config:
+        config.write(user_config)
+    print(f"The file was created at this path {CONIFGPATH}")
+    show_config()
+
+
+def show_config():
+    """Show the configs to the user"""
+
+    # read the config
+    parser = ConfigParser()
+    parser.read(f"{CONIFGPATH}")
```

Comment:
Use the variable directly:

```suggestion
    parser.read(CONIFGPATH)
```